frontend/src/Student/components/OtherMediaSubmissionForm.js

```diff
@@ -30,10 +30,6 @@ class OtherSubmissionForm extends Component {
     user: PropTypes.shape({
       username: PropTypes.string
     }).isRequired,
-    forShow: PropTypes.shape({
```

Comment:
If you're removing this in each of these files, you need to add in the `propTypes` for `data`